README.md

```diff
@@ -111,7 +111,7 @@ async def main():
 
 
 if __name__ == "__main__":
-    asyncio.get_event_loop().run_until_complete(main())
+    asyncio.run(main())
 ```
 
 The line `if __name__ == "__main__":` is required on Windows and macOS due to the way the `multiprocessing` library works. It is not required on Linux, but it is recommended. It is omitted from the rest of the examples below.
```

Comment:
Add mention about Python 3.9 and asyncio.run:

```md
Due to limitations in the RuuviTag package's Bleak adapter, Python 3.10 or later is required to use asyncio.run(main()). If using Python 3.9, replace asyncio.run(main()) with asyncio.get_event_loop().run_until_complete(main()).
```

Reply:
Added.